lib/AST/Decl.cpp

```diff
@@ -2824,6 +2824,13 @@ static bool checkAccess(const DeclContext *useDC, const ValueDecl *VD,
   auto access = getAccessLevel();
   auto *sourceDC = VD->getDeclContext();
 
+  // Preserve "fast path" behavior for everything outside of
```

Comment:
I think it's the other way around, now you're only using the fast path below for protocol extension members and operators

Reply:
What I understood from @jrose-apple that's how it's supposed to be - preserve "fast path" behavior only for protocols and operators, and everything else is supposed to use scopes?

Reply:
Ah comment!